BigBaseV2/src/backend/looped/self/seat_belt.cpp

```diff
@@ -0,0 +1,24 @@
+#include "backend/looped/looped.hpp"
+
+namespace big
+{
+	static bool bLastSeatBelt = false;
+
+	void looped::self_seat_belt()
```

Comment:
I thought we already had seat belt under Vehicle ?

Reply:
my bad, didnt check vehicle view